boards/st/sensortile_box_pro/doc/index.rst

```diff
@@ -219,61 +219,44 @@ Console
 
 There are two possible options for Zephyr console output:
 
-- through UART4 which is available on SWD connector (JP2). In this case a JTAG adapter
-  can be used to connect SensorTile.box PRO and have both SWD and console lines available.
-
-  To enable console and shell over UART
-
-  - switch the console lines from cdc_acm to uart4
-    (:file:`boards/st/sensortile_box_pro/sensortile_box_pro.dts`)
-
-  - comment out the USB configuration macros
-    (:file:`boards/st/sensortile_box_pro/sensortile_box_pro_defconfig`)
+- through USB as USB CDC/ACM class. This is the default case present in the board dts file
+  and enabled thanks :kconfig:option:`CONFIG_BOARD_SERIAL_BACKEND_CDC_ACM`.
 
 .. code-block:: dts
    :caption: boards/st/sensortile_box_pro/sensortile_box_pro.dts
 
    / {
        chosen {
-          zephyr,console = &uart4;
-          zephyr,shell-uart = &uart4;
-          //zephyr,console = &cdc_acm_uart0;
-          //zephyr,shell-uart = &cdc_acm_uart0;
+          zephyr,console = &cdc_acm_uart0;
+        };
+     };
+
+     &zephyr_udc0 {
+        cdc_acm_uart0: cdc_acm_uart0 {
+                compatible = "zephyr,cdc-acm-uart";
         };
      };
 
-.. code-block:: Kconfig
-   :caption: boards/st/sensortile_box_pro/sensortile_box_pro_defconfig
 
-   # Comment out following USB config lines when
-   # switching console to UART
-   #CONFIG_USB_DEVICE_STACK=y
-   #CONFIG_USB_DEVICE_VID=0x0483
-   #CONFIG_USB_DEVICE_PID=0x1235
-   #CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC SensorTile.box PRO"
-   #CONFIG_USB_CDC_ACM_LOG_LEVEL_OFF=y
-   #CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT=n
+- through UART4 which is available on SWD connector (JP2). In this case a JTAG adapter
+  can be used to connect SensorTile.box PRO and have both SWD and console lines available.
 
+  To enable console and shell over UART.
 
-- through USB as USB CDC/ACM class. This is the default case present in the board dts file.
+  - in your prj.conf overwrite the configuration :code:`CONFIG_BOARD_SERIAL_BACKEND_CDC_ACM=n`
+
+  - add an overlayfile named ``<board>.overlay``:
```

Comment:
```suggestion
  - add an overlay file named ``<board>.overlay``:
```
Minor typo. Both modifications are needed to enable serial console, right?
FInal question: I have seen some applications which disable https://docs.zephyrproject.org/latest/kconfig.html#CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT, in order to disable default USB usage (typically for console) and add their own USB configuration.
For example: https://github.com/zephyrproject-rtos/zephyr/blob/0781caf408459dfd6c91477cedb62a43912a8c51/samples/bluetooth/hci_usb/prj.conf
I guess, this is working also with your modifications.